dynamic-config/config-nacos/src/main/java/org/apache/servicecomb/config/archaius/sources/NacosConfigurationSourceImpl.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.config.archaius.sources;
+
+import static com.netflix.config.WatchedUpdateResult.createIncremental;
+import static org.apache.servicecomb.config.client.ConfigurationAction.CREATE;
+import static org.apache.servicecomb.config.client.ConfigurationAction.DELETE;
+import static org.apache.servicecomb.config.client.ConfigurationAction.SET;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+import javax.validation.constraints.NotNull;
+
+import com.alibaba.nacos.api.NacosFactory;
+import com.alibaba.nacos.api.config.ConfigService;
+import org.apache.commons.configuration.Configuration;
+import org.apache.servicecomb.config.ConfigMapping;
+import org.apache.servicecomb.config.client.NacosClient;
+import org.apache.servicecomb.config.client.NacosConfig;
+import org.apache.servicecomb.config.client.ConfigurationAction;
+import org.apache.servicecomb.config.spi.ConfigCenterConfigurationSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.ImmutableMap;
+import com.netflix.config.WatchedUpdateListener;
+import com.netflix.config.WatchedUpdateResult;
+
+public class NacosConfigurationSourceImpl implements ConfigCenterConfigurationSource {
+  private static final Logger LOGGER = LoggerFactory.getLogger(ConfigCenterConfigurationSource.class);
+
+  private final Map<String, Object> valueCache = new ConcurrentHashMap<>();
+
+  private List<WatchedUpdateListener> listeners = new CopyOnWriteArrayList<>();
+
+  private static final String NACOS_CONFIG_SERVER_ADDR_KEY = "nacos.config.serverAddr";
```

Comment:
Can use the same constant in NacosConfig

Reply:
The name of key is replaced with "SERVER_ADDR".